packages/components/src/__future__/Tag/_docs/RemoveableTag.stories.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { v4 as uuidv4 } from "uuid"
```

Comment:
We should now be using `useId` from React.
But given we are looking at stories, just give them ids that make sense instead of using this.